addOns/commonlib/src/main/resources/org/zaproxy/addon/commonlib/internal/vulns/vulnerabilities.xml

```diff
@@ -1069,13 +1081,13 @@ A Web application should invalidate a session after a predefined idle time has p
 6. **Security Headers:** Utilize security headers (e.g., Content Security Policy) to control how browsers and search engines interact with your website.
 
 A proactive and layered approach to security is crucial for safeguarding against Insecure Indexing.</solution>
-	<reference>http://projects.webappsec.org/Insecure-Indexing</reference>
+<reference/>
```

Comment:
Would be good to keep a reference.

Reply:
I really didn't find a decent resource for this, will look again.